### PR TITLE
Add support to Docker Compose v2

### PR DESCRIPTION
**Pull request by noliveleger**
# Description
kobo-install uses compatible command for `docker-compose` (v1) and `docker compose` (v2). 
It detects if `docker-compose` is present on the system. Otherwise, it falls back to `docker compose`.

## Additional info
On macOS. Docker Desktop creates a symlink on `docker-compose` then runs `docker compose` behind the scene. 
kobo-install will see at v1 even v2 is checked in preferences. But it does not matter, it still works that way. (tested with Docker Desktop for mac `4.14.0` and Compose: `v2.12.2`).

### CODE REVIEW

File: helpers/upgrading.py [hunk 1]
```diff
@@ -146,7 +148,24 @@ def new_terminology(upgraded_dict):
...
+        try:
+            upgraded_dict['compose_version']
+        except KeyError:
+            # FIXME On macOS, Docker Desktop always installs a symlink for
+            #   `docker-compose`. Version will be always detected as v1 even if
+            #   user has chosen v2 in Docker Desktop preferences.
+            if which('docker-compose') is None:
+                compose_version = 'v2'
+            else:
+                compose_version = 'v1'
```
Comment (joshuaberetta) [inline helpers/upgrading.py:161]: Is there is a reason for using `try...except` here? Maybe something more compact:
```python3
if 'compose_version' not in upgraded_dict:
    compose_version = 'v2' if which('docker-compose') is None else 'v1'
```

File: helpers/utils.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+def run_docker_compose(config_dict: dict, command: list[str]) -> list[str]:
+
+    try:
+        compose_version = config_dict['compose_version']
+    except KeyError:
+        # Consider version as v1. If 'compose_version' key is missing, it is
+        # probably because user has pulled kobo-install without running `--update`
+        compose_version = 'v1'
```
Comment (joshuaberetta) [inline helpers/utils.py:11]: Maybe something simpler like this?
```python
compose_version = config_dict.get('compose_version', 'v1')
```

**Review by joshuaberetta — CHANGES_REQUESTED** [review]
Tested on Ubuntu 22.04 LTS and works as expected 💪

File: helpers/upgrading.py (hunk 1 above)
Comment (noliveleger) [reply helpers/upgrading.py]: There is a reason, but not a good one :-P

**Review by joshuaberetta — APPROVED** [review]

**Pull request merged**